lib/linez.spec.ts

```diff
@@ -182,5 +198,156 @@ describe('linez', () => {
 			});
 
 		});
+
+		describe('code block support', () => {
+
+			it('multiline comments api', () => {
+				var doc = linez([
+					'foo',
+					'/**',
+					' * foo',
+					' * bar',
+					' */',
+					'bar',
+					'    /*',
+					'     * foo',
+					'     * bar',
+					'     */',
+				].join('\n'));
+				expect(doc.lines[0].text).to.eq('foo');
+				var commentBlock1 = doc.lines[1].block.multilineComment;
+				expect(commentBlock1).to.have.length(4);
+				expect(commentBlock1[0].text).to.eq('/**');
+				expect(commentBlock1[1].text).to.eq(' * foo');
+				expect(commentBlock1[2].text).to.eq(' * bar');
+				expect(commentBlock1[3].text).to.eq(' */');
+				var commentBlock2 = doc.lines[6].block.multilineComment;
+				expect(commentBlock2).to.have.length(4);
+				expect(commentBlock2[0].text).to.eq('    /*');
+				expect(commentBlock2[1].text).to.eq('     * foo');
+				expect(commentBlock2[2].text).to.eq('     * bar');
+				expect(commentBlock2[3].text).to.eq('     */');
+			});
+
+			it('format multiline comments', () => {
+				var doc = linez([
+					'    /**',
+					'    * foo',
+					'    * bar',
+					'    */',
+					'/**',
+					'    * foo',
+					'    * bar',
+					'    */',
+					'*/',
+				].join('\n'));
+				doc.lines.forEach(line => {
+					var multilineComment = line.block.multilineComment;
+					var prefix = "";
+					if (multilineComment) {
+						if (/^(\s+)/.test(line.text)) {
+							prefix = RegExp.$1;
+						}
+						multilineComment.forEach(line => {
+							if (/^\s*\*/.test(line.text)) {
+								line.text = prefix + " " + line.text.trim();
+							}
+						});
+					}
+				});
+
+				expect(doc.toString()).to.eq([
+					'    /**',
+					'     * foo',
+					'     * bar',
+					'     */',
+					'/**',
+					' * foo',
+					' * bar',
+					' */',
+					'*/',
+				].join('\n'));
+			});
+
+			it('format javascript multiline string', () => {
```

Comment:
Again, this test is too focused on implementation details. You should only be checking that the API is structuring the lines the way you expect, not doing any kind of formatting after the fact.